perf/src/sigverify.rs

```diff
@@ -608,15 +610,38 @@ pub fn shrink_batches(batches: &mut Vec<PacketBatch>) {
 
 pub fn ed25519_verify_cpu(batches: &mut [PacketBatch], reject_non_vote: bool, packet_count: usize) {
     debug!("CPU ECDSA for {}", packet_count);
-    PAR_THREAD_POOL.install(|| {
-        batches.into_par_iter().for_each(|batch| {
-            batch.par_iter_mut().for_each(|packet| {
-                if !packet.meta.discard() && !verify_packet(packet, reject_non_vote) {
-                    packet.meta.set_discard(true);
-                }
-            })
+
+    let thread_count = packet_count
+        .saturating_add(VERIFY_MIN_PACKETS_PER_THREAD)
```

Comment:
If we're trying to find the ceiling here, should this be `VERIFY_MIN_PACKETS_PER_THREAD-1`? I.e. at `packet_count = 16` we only need 1 thread, not 2

Reply:
That's true. I originally had it like that and removed it when clippy complained about integer arithmetic. The net effect is that we get an extra thread 1 packet sooner *shrug*